site2/docs/client-libraries-go.md

```diff
@@ -302,6 +302,80 @@ fmt.Println(msg.Payload())
 canc()
 ```
 
+#### How to use Prometheus metrics in producer
+
+Pulsar Go client registers client metrics using Prometheus. In this guide, we'll show you how to create a simple Pulsar producer application that exposes Prometheus metrics via HTTP.
```

Comment:
```suggestion
Pulsar Go client registers client metrics using Prometheus. This section demonstrates how to create a simple Pulsar producer application that exposes Prometheus metrics via HTTP.
```

Reply:
Reason for this change: https://docs.google.com/document/d/1lc5j4RtuLIzlEYCBo97AC8-U_3Erzs_lxpkDuseU0n4/edit#bookmark=id.gfwms5zehcak